app/assets/stylesheets/custom.css

```diff
@@ -0,0 +1,19 @@
+legend,
+fieldset,
+select,
+textarea,
+input,
+button {
+  background-color: transparent;
+  border: none;
+  padding: 0;
+  margin: 0;
+  overflow:hidden;
+}
+
+.hero_bgImage
+{
+    background: linear-gradient(rgba(96, 165, 250, 0.9),rgba(129, 140, 248, 0.9)),url("../images/heroImage.jpg");
```

Comment:
It seems we do not have such image in repo.
First folder under `assets` is ignored so the correct path is `url("heroImage.jpg")`

Reply:
Interesting. Why are subfolders ignored when giving a path? Image will be added in next commit.